ts/features/itwallet/issuance/screens/ItwIssuanceCredentialTrustIssuerScreen.tsx

```diff
@@ -140,8 +142,22 @@ const ContentView = ({ credentialType, eid }: ContentViewProps) => {
       } as RequiredClaim)
   );
 
+  const mixPanelCredential = useMemo(
+    () => CREDENTIALS_MAP[credentialType],
+    [credentialType]
+  );
+
+  const trackScrollToBottom = (crossed: boolean) => {
+    if (crossed) {
+      trackIssuanceCredentialScrollToBottom(
+        mixPanelCredential,
+        ITW_ROUTES.ISSUANCE.CREDENTIAL_TRUST_ISSUER
+      );
+    }
+  };
+
```

Comment:
Why use memo instead of getting the value directly in the function? To we need the value elsewhere?

```suggestion
  const trackScrollToBottom = (crossed: boolean) => {
    if (crossed) {
      trackIssuanceCredentialScrollToBottom(
        CREDENTIALS_MAP[credentialType],
        ITW_ROUTES.ISSUANCE.CREDENTIAL_TRUST_ISSUER
      );
    }
  };

```

Reply:
I had thought about the fact that the `credentialType` could change but thinking about it this is not the case.
Fixed, thanks!